guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -91,6 +91,19 @@ If set to `Yes`, public SSH keys will be installed on the registered host.
 The inherited value is based on the `host_registration_remote_execution` parameter.
 It can be inherited, for example from a host group, an operating system, or an organization.
 When overridden, the selected value will be stored on host parameter level.
+ifdef::katello[]
+. From the *REX pull mode* list, select whether you want to deploy {Project} remote execution pull client or not.
++
+If set to `Yes`, {Project} remote execution pull client will be installed on the registered host.
```

Comment:
```suggestion
If set to `Yes`, {Project} remote execution pull client is installed on the registered host.
```
We avoid future tense and live in a continuous present according to IBM Style :D